.github/workflows/ci.yml

```
@@ -32,7 +32,17 @@ jobs:
         if: steps.cache-toolchain.outputs.cache-hit != 'true'
         run: make arm_sdk_install
 
+      - name: Hydrate the PICO sdk
+        description: Hydration is required to build the PICO tool
```

actionlint (1.7.7)

unexpected key "description" for "step" section. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "uses", "with", "working-directory"

(syntax-check)